### DISCUSSION TITLE
Wrong python version

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-app#2970] [Q&A] [answered]
Hi there,
I have just installed Termux, trying to have a fully wokring python with jupyter-lab over a Samsung Galaxy Tab S8 Ultra.
I have installed all the modules, matplotlib, pandas, numpy, all working, now i'm having problem with scipy, so i added the pointless repo, to execute pkg install scipy, and it worked, the problem is that is not working the import scipy.
By explore the folder, i find out scipy was installed inside the folder python3.7, but i'm actually using python3.10 how can i solve this? Thanks to anyone.

**Comment by 2096779623** [comment, marked as answer]
termux-packages does not provide older versions of `python`, try to install scipy with the following command:
```
curl -LO https://its-pointless.github.io/setup-pointless-repo.sh bash setup-pointless-repo.sh
pkg i python build-essential clang gcc-11
setupclang-gfort-11
pkg i openblas
export BLAS=/data/data/com.termux/files/usr/lib/libblas.so
export LAPACK=/data/data/com.termux/files/usr/lib/liblapack.so
export CC=clang
export CPP=clang++
pip install wheel
pip install scipy
```

* Remove the old version of python first.
